db/migrate/20200609082714_adds_documentable_to_documents.rb

```diff
@@ -0,0 +1,5 @@
+class AddsDocumentableToDocuments < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :documents, :documentable, polymorphic: true, index: true
```

Comment:
Foreign key isn't an option for polymorphic relationships

Reply:
👍